.changelog/15727.txt

```diff
@@ -0,0 +1,3 @@
+```release-note:note
+dependency: Upgrade to use Go 1.18.9.
+```
```

Comment:
I don't really have a preference but the last one I remember doing was formatted like this: https://github.com/hashicorp/consul/blob/main/.changelog/12808.txt#L1-L4